staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc_test.go

```diff
@@ -150,6 +150,77 @@ func TestClientCache(t *testing.T) {
 	assertCacheLen(t, cache, 2)
 }
 
+func TestNewClient(t *testing.T) {
+	tests := []struct {
+		name         string
+		config       map[string]string
+		wantErr      bool
```

Comment:
Change all the `wantErr`s to a string and compare the actual error message

Reply:
Good idea. Thanks for your suggestion.